prime-angular-frontend/src/app/modules/adjudication/shared/components/site-registration-tabs/site-registration-tabs.component.ts

```diff
@@ -63,9 +63,11 @@ export class SiteRegistrationTabsComponent implements OnInit {
 
   public communityPracticeColumns: string[];
   public communityPharmacyColumns: string[];
+  public siteTabIndex: number;
 
-  private careSettingCode: CareSettingEnum;
   private routeUtils: RouteUtils;
+  private tabIndexCareSettingMap: Record<number, CareSettingEnum>;
+  private careSettingTabIndexMap: { [key in CareSettingEnum]?: number }
```

Comment:
Would `tabIndexToCareSettingMap` and `careSettingToTabIndexMap` be better naming?